packages/@coorpacademy-components/src/template/battle/index.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Provider from '../../atom/provider';
+import Cta from '../../atom/cta';
+import Link from '../../atom/link';
+import Avatar from '../../atom/avatar';
+import style from './style.css';
+
+function backgroundImage(url) {
+  return {
+    background: `linear-gradient(
+      rgba(0, 0, 0, 0.55), 
+      rgba(0, 0, 0, 0.55)
+      ),url(${url})`,
+    backgroundSize: 'cover'
+  };
+}
+
+const Button = props => {
+  return (
+    <div>
+      <Cta
+        className={`${props.className} ${style[props.buttonColor]}`}
+        submitValue={props.submitValue}
+        onClick={props.onClick}
+        secondary={props.secondary}
+      />
+    </div>
+  );
+};
+const App = (props, context) => {
+  const {translate} = context;
+  const {
+    name,
+    courseName,
+    submitReviewCourseValue,
+    submitStartBattleValue,
+    levelName,
+    onClickButtonBattle,
+    onClickButtonCourse,
+    urlAvatar,
+    urlImg
+  } = props;
+  const challengeLabel = translate('challenges you on');
```

Comment:
```suggestion
  const challengeLabel = translate('challenges you on', {name:name, courseName: courseName});
```
`challenges you on`: `<p><span>{{name}} </span>challenges you on </p><h3>{{courseName}}</h3>`